seed/challenges/01-front-end-development-certification/basic-javascript.json

```diff
@@ -3345,7 +3345,7 @@
       "tests": [
         "assert(code.match(/else/g).length > 3, 'message: You should have at least four <code>else</code> statements');",
         "assert(code.match(/if/g).length > 3, 'message: You should have at least four <code>if</code> statements');",
-        "assert(code.match(/return/g).length >= 5, 'message: You should have at least five <code>return</code> statements');",
+        "assert(code.match(/return/g).length >= 1, 'message: You should have at least one <code>return</code> statements');",
```

Comment:
@erictleung You are going to probably kill me for this but I have one more request:
`statements` > `statement`